README.md

```diff
@@ -147,12 +147,29 @@ Writing & watching frontend tests
 ```bash
 # Run Golang tests using sqlite3 as database (default)
 go test ./pkg/...
+```
+
+##### Running the MySQL or Postgres backend tests:
+
+Run these by setting `GRAFANA_TEST_DB` in your environment.
+
+- `GRAFANA_TEST_DB=mysql` to test MySQL
+- `GRAFANA_TEST_DB=postgres` to test Postgres
+
+Follow the instructions in `./devenv` to spin up test containers running the appropriate databases with `docker-compose`
+- Use `docker/blocks/mysql_tests` or `docker/blocks/postgres_tests` as appropriate
+
+```bash
+# Tests can only be ran in one Go package at a time due to clashing db queries.
+GRAFANA_TEST_DB=mysql go test ./pkg/services/sqlstore/...
+
+# Or run all the packages in a loop:
+for d in $(go list ./pkg/...); do GRAFANA_TEST_DB=mysql go test $d; done
 
-# Run Golang tests using mysql as database - convenient to use /docker/blocks/mysql_tests
-GRAFANA_TEST_DB=mysql go test ./pkg/...
+# Postgres
+GRAFANA_TEST_DB=postgres go test ./pkg/services/sqlstore/...
 
-# Run Golang tests using postgres as database - convenient to use /docker/blocks/postgres_tests
-GRAFANA_TEST_DB=postgres go test ./pkg/...
+for d in $(go list ./pkg/...); do GRAFANA_TEST_DB=postgres go test $d; done
```

Comment:
It's a lit bit confusing, maybe it should be split in two different blocks (MySQL and Postgres)